lib/git-revision-view.coffee

```diff
@@ -129,6 +114,10 @@ class GitRevisionView
       editor1: newTextEditor    # the older revision
       editor2: editor           # current rev
 
+    SplitDiff.setConfig 'leftEditorColor', 'green'
+    SplitDiff.setConfig 'leftEditorColor', 'red'
```

Comment:
> should this ^^ be rightEditorColor?

Yes. I see you already merged this. Were you planning on correcting this, or would you like me to put up another PR?